docs/componentExtension.md

```diff
@@ -0,0 +1,72 @@
+## Customizing and Extending Components
+
+As mentioned in the Architecture.md when customizing or extending the sample app component, do not modify the packages within `@sfcc-bff` and `@sfcc-core`. These packages will be published and consumed via NPM. Instead, create a new custom package within the monorepo that registers itself with `@sfcc-core` and provides access to data from a third-party service.
+
+This example below illustates how to create a product recommendations extension for product details component:
+
+Background : Product Details Page displays product name, item id, color swatches, images, price etc, our goal is extend this component to display this product's recommendations on Product Detail Page. 
```

Comment:
I think it's better to say "product id" than "item id".  Missing comma after "price". "etc" should have a period at the end.  "our goal is extend" should be "Our goal is to extend".
Maybe @pdisabito can provide more feedback.

Reply:
thank you! I fixed it. I am asking Paul to review it tool